regression/test.pl

```diff
@@ -211,13 +216,13 @@ ($$$$)
   -c CMD     run tests on CMD - required option
   -i <regex> options in test.desc matching the specified perl regex are ignored
   -j <num>   run <num> tests in parallel (requires Thread::Pool::Simple)
+  -d         dry-run: print the tests that would be run, but don't actually run them
   -h         show this help and exit
   -C         core: run all essential tests (default if none of C/T/F/K are given)
   -T         thorough: run expensive tests
   -F         future: run checks for future features
   -K         known: run tests associated with known bugs
 
-
```

Comment:
Not sure this change is essential, but I'll leave it to you whether you feel this makes the message look neater.

Reply:
Oops, undone